.github/workflows/CI.yaml

```
@@ -203,9 +203,11 @@ jobs:
     strategy:
       matrix:
         helm: [true, false]
+        scheduling_gates: [true, false]
     with:
       runtime_tag: dev-${{ needs.prepare_ci_run.outputs.DATETIME }}
       helm-install: ${{ matrix.helm }}
+      scheduling_gates: ${{ matrix.scheduling_gates }}
```

please use `-` for the variable names :)
```suggestion
      scheduling-gates: ${{ matrix.scheduling_gates }}
```